src/View/Cell.php

```diff
@@ -259,15 +260,17 @@ protected function _cacheConfig($action)
      * This is because PHP will not allow a __toString() method to throw an exception.
      *
      * @return string Rendered cell
+     * @throws \Error Include error details for PHP 7 fatal errors.
      */
     public function __toString()
     {
         try {
             return $this->render();
         } catch (Exception $e) {
             trigger_error(sprintf('Could not render cell - %s [%s, line %d]', $e->getMessage(), $e->getFile(), $e->getLine()), E_USER_WARNING);
-
             return '';
+        } catch (Error $e) {
+            throw new Error(sprintf('Could not render cell - %s [%s, line %d]', $e->getMessage(), $e->getFile(), $e->getLine()));
```

Comment:
I like the ToStringTrait idea, but I searched the entire source code for another toString that had a try/catch block and this was the only one.

You've made a mistake by killing the PHP process in response to none fatal `Exception` errors. Those are currently rendered as warnings in the output HTML and the PHP thread continues to render the rest of the response. Only `Error` are fatal server 500 responses.

I think calling the error handler directly obfuscates the issue you are trying to fix. If I came to this source code without reading this issue. I would not know why this was being done. It would be confusing to me.

I think the original PR can be read by other programmers and makes the issue clear as to what is being fixed.

All of this discussion just illustrates my third recommendation in this PR. Stop using `__toString` because it makes debugging more difficult when `$cell->render()` makes everything here irrelevant.

The other option could be to make the `View::cell()` method return the rendered result of the cell. Just like the `View::element()` method. Why do people need the cell object in views? We could just add another method called `View::createCell()` to get the object reference.